src/runtime/cuda/cuda_queue.cpp

```diff
@@ -47,12 +47,61 @@
 
 #include <cassert>
 #include <memory>
+#include <vector>
 
 namespace hipsycl {
 namespace rt {
 
 namespace {
 
+
+unsigned select_ptx_version(unsigned sm_version, unsigned& ptx_target) {
+  // Our bitcide libraries need at least ptx +60
+  unsigned ptx_version = 60;
+  
+  // for each sm, stores minimum ptx version required
+  // based on data from
+  // https://docs.nvidia.com/cuda/parallel-thread-execution/index.html#ptx-module-directives-target
+  static std::vector<std::array<unsigned, 2>> sm_min_ptx_version =
+      {{72, 61},
+       {75, 63},
+       {80, 70},
+       {86, 71},
+       {87, 74},
+       {89, 78},
+       {90, 78},
```

Comment:
If we care about 90a (which looks like a good choice for JIT, since we don't care about future-compatibility of PTX):

```suggestion
       {90, 80},
```